.github/workflows/fast_testing.yaml

```diff
@@ -35,4 +35,17 @@ jobs:
       - name: Clone the module
         uses: actions/checkout@v2
 
-      - run: make test
+      - name: Cache rocks
+        uses: actions/cache@v2
+        id: cache-rocks
+        with:
+          path: .rocks/
+          key: cache-rocks-${{ matrix.tarantool }}-01
+
+      - name: Install requirements
+        run: |
+          tarantoolctl rocks install luatest 0.5.0
+        if: steps.cache-rocks.outputs.cache-hit != 'true'
```

Comment:
You are enabled luatest in CI, but luatest is unused. You add luatest-style tests later in further commits.

Reply:
Fixed. Luatest appears in ci along with the first luatest's tests